testing/testcafe/tests/htmlEditor/format.ts

```diff
@@ -1,13 +1,17 @@
 import SelectBox from '../../model/selectBox';
 import url from '../../helpers/getPageUrl';
 import createWidget from '../../helpers/createWidget';
+import { clearTestPage } from '../../helpers/clearPage';
 
-fixture`HtmlEditor - formats`
-  .page(url(__dirname, '../container.html'));
+fixture.disablePageReloads`HtmlEditor - formats`
+  .page(url(__dirname, '../containerQuill.html'))
+  .afterEach(async () => clearTestPage());
```

Comment:
Question.

Does this ``` afterEach ``` in the fixture work properly?
I had some troubles with it when developing the ```safeSizeTest``` helper.
It had not been executed in some test cases.

But maybe I missed something.

Reply:
Yes, afterEach works properly in this case